docs/docs/Configuration/configuration-custom-database.mdx

```diff
@@ -4,174 +4,154 @@ slug: /configuration-custom-database
 ---
 Langflow's default database is [SQLite](https://www.sqlite.org/docs.html), but you can configure Langflow to use PostgreSQL instead.
 
-This guide walks you through setting up an external database for Langflow by replacing the default SQLite connection string `sqlite:///./langflow.db` with PostgreSQL.
+This guide walks you through setting up an external database for Langflow by replacing the default SQLite connection string `sqlite:///./langflow.db` with PostgreSQL, both in local and containerized environments.
 
-## Prerequisite
+In this configuration, all structured application data from Langflow including flows, message history, and logs, is instead managed by PostgreSQL.
+PostgreSQL is better suited for production environments due to its robust support for concurrent users, advanced data integrity features, and scalability.
+Langflow can more efficiently handle multiple users and larger workloads by using PostgreSQL as the database.
 
-* A [PostgreSQL](https://www.pgadmin.org/download/) database
+## Prerequisites
 
-## Connect Langflow to PostgreSQL
+- [Install and start Langflow](/get-started-installation)
+- A [PostgreSQL](https://www.pgadmin.org/download/) database
 
-1. If Langflow is running, quit Langflow.
+## Connect Langflow to a local PostgreSQL database
+
+1. If Langflow is running, stop Langflow with <kbd>Ctrl+C</kbd>.
 
 2. Find your PostgreSQL database's connection string in the format `postgresql://user:password@host:port/dbname`.
 
-  The hostname in your connection string depends on how you're running PostgreSQL:
+    The hostname in your connection string depends on how you're running PostgreSQL:
 
-  - If you're running PostgreSQL directly on your machine, use `localhost`.
-  - If you're running PostgreSQL in Docker Compose, use the service name, such as `postgres`.
-  - If you're running PostgreSQL in a separate Docker container with `docker run`, use the container's IP address or network alias.
+    - If you're running PostgreSQL directly on your machine, use `localhost`.
+    - If you're running PostgreSQL in Docker Compose, use the service name, such as `postgres`.
+    - If you're running PostgreSQL in a separate Docker container with `docker run`, use the container's IP address or network alias.
+    - If you're running a cloud-hosted PostgreSQL, your provider will share your connection string, which includes a username and password.
 
-3. Create a Langflow `.env` file if you don't already have one:
+3. Create a Langflow `.env` file.
```

Comment:
```suggestion
3. Create a Langflow `.env` file:
```

When the instruction immediately precedes a code block that demonstrates the instruction, use a colon.